src/card.tsx

```diff
@@ -282,11 +282,14 @@ const CardActionPauseIcon = ({color}: IconProps) => <IconPauseFilled color={colo
 
 const CardActionPlayIcon = ({color}: IconProps) => <IconPlayFilled color={color} size={12} />;
 
-const useVideoWithControls = (
+export const useVideoWithControls = (
```

Comment:
should we abstract this in a separate file instead? It's not something specific to cards anymore

Reply:
Yes I thought about it, I can extract it if you prefer

Reply:
Yes, I think we should do it in order to keep the code well organized

Reply:
I think we could create a `top-actions.tsx` file with all the top actions components/hooks including this one, what do you think?

Reply:
talked offline, we'll move this refactor to the future (when we have over media variant)